llmops/common/deployment/provision_deployment.py

```diff
@@ -102,7 +104,6 @@
         if stage == elem["ENV_NAME"]:
             endpoint_name = elem["ENDPOINT_NAME"]
             deployment_name = elem["CURRENT_DEPLOYMENT_NAME"]
-            deployment_conda_path = elem["DEPLOYMENT_CONDA_PATH"]
             deployment_base_image = elem["DEPLOYMENT_BASE_IMAGE_NAME"]
```

Comment:
If we don't want to give poosibility to use base docker images then these variable and config parameter are not needed.

Reply:
fixed